doc/contributing.rst

```diff
@@ -1 +1,9 @@
 .. include:: ../CONTRIBUTING.rst
+
+Performance tests
+=================
+
+The test suite contains performance tests, which are disabled by the default options given in :file:`setup.cfg`.
+To run these, use::
+
+    pytest … --benchmark-only …
```

Comment:
Trying to run this returns the following error.
```
C:\Repo\ixmp>pytest -k 'perf' --benchmark-only
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --benchmark-skip --benchmark-only
  inifile: C:\Repo\ixmp\setup.cfg
  rootdir: C:\Repo\ixmp
```

Reply:
In order to use this feature `pytest-benchmark` needs to be installed. Should this then not become part of the package requirements?